### DISCUSSION TITLE
[Ultima][Ultima-React] Is there a version without next.js?

### DISCUSSION THREAD

**Discussion by allanfreitas** [primefaces/community#195] [PrimeReact Templates] [answered]
We need to use UltimaReact on full SPA mode without next.js for a SPA of one of our portal modules.

Do you provide any way to do this?

**Comment by habubey** [comment, marked as answer]
We are sorry that we do not provide such a service.
Additionally, the majority of our templates are built in NextJS (app router), while a limited number remain in CRA mode. These few templates are currently being migrated to NextJS as soon as possible.

**Comment by cagataycivici** [comment]
As far as I remember, Next.js has configuration to disable SSR to switch to SPA mode. In the past all our templates were CRA but Next.JS is currently the suggested one in React community so migrating them to Next.js one by one.

**Reply by allanfreitas** [reply]
I couldn't find a way to properly use it for dynamic routing for websystem, I have no experience with Next.js, and I couldn't find it in the documentation or searching the internet. Everything points to the export, but it doesn't seems to work for dynamic routes as properly SPA.

**Reply by mertsincan** [reply]
Maybe, this link can help to you to disable SSR;
https://github.com/vercel/next.js/discussions/17863
